app/utils/token-info.ts

```diff
@@ -123,8 +131,9 @@ export async function getFullTokenInfo(
         logoURI: sdkTokenInfo.logoURI ?? undefined,
         name: sdkTokenInfo.name,
         symbol: sdkTokenInfo.symbol,
-        tags
-    }
+        tags,
+        verified: !!(legacyCdnTokenInfo || sdkTokenInfo.verified),
```

Comment:
This means ‘if you're on the legacy token list, or the UTL registry said you're verified, you're verified.’ Is that sound, @mcintyre94?

Reply:
Sorry, I missed this PR before! I think this is reasonable, but if the UTL registry contains the same token as unverified maybe that should take precedence?

So:
- Not on the legacy token list at all, UTL verified determines it
- On legacy token list and UTL, UTL verified determines it
- On legacy token list and not UTL, verified

Reply:
So, if I understand correctly, that basically implies that we should do this?
```suggestion
        verified: sdkTokenInfo.verified ?? false,
```

1. By this line, we know that `sdkTokenInfo` is non-null.
2. The ‘not on UTL but on legacy list’ case is handled above.

Reply:
Building a Vercel preview for that now: #289.